### PR TITLE
refactor(zopafooter): update with new legal copy

### PR DESCRIPTION
**Pull request by endymion1818**
re #65

Footer refactored to resemble www.zopa.com:

- Links will be the same globally
- Icons updated to match global style
- Instagram icon & link added
- Footer legal text updated
- Layout reconfigured to match global style

### CODE REVIEW

File: src/components/molecules/ZopaFooter/Legal/__snapshots__/Legal.test.tsx.snap [hunk 1]
```diff
@@ -24,23 +19,21 @@ exports[`<Legal /> renders the component 1`] = `
...
-    class="c2"
-  />
-  <div
-    class="c0"
-  >
+    <br />
+    <p
+      class="c1"
+    >
+      © Zopa Bank Limited 2016 All rights reserved. 'Zopa' is a trademark of Zopa Bank Limited.
```
Comment (spadger) [inline src/components/molecules/ZopaFooter/Legal/__snapshots__/Legal.test.tsx.snap]: 2016?

File: src/components/molecules/ZopaFooter/Legal/Legal.tsx [hunk 2]
```diff
@@ -13,24 +12,24 @@ const StyledParagraph = styled.p`
...
   };
 
   return (
     <>
       <Wrapper>
         <StyledParagraph>{about}</StyledParagraph>
-      </Wrapper>
-      <Separator />
-      <Wrapper>
+        <br />
```
Comment (lluia) [inline src/components/molecules/ZopaFooter/Legal/Legal.tsx]: I think we could avoid `<br />` and instead add some CSS to `<StyledParagraph />` to have _new lines_ 🙃

Comment (poteirard) [reply src/components/molecules/ZopaFooter/Legal/Legal.tsx]: This is just to change the text. After this, a new PR will come with a refactor. That's why we allowed this <br>

Comment (endymion1818) [reply src/components/molecules/ZopaFooter/Legal/Legal.tsx]: This has been done, thanks Lluis 👍

File: src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx [hunk 3]
```diff
@@ -14,19 +15,24 @@ export interface ISocialLinkData {
...
+  {
+    component: Twitter,
+    href: 'https://twitter.com/zopa',
+    label: 'twitter',
+  },
 ];
 
 const SocialLinks = () => (
-  <FlexRow>
+  <FlexRow justify={'flex-end'}>
```
Comment (poteirard) [inline src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: justify="flex-end"

Comment (endymion1818) [reply src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx]: 👍

File: .gitignore [hunk 4]
```diff
@@ -10,3 +10,7 @@ dist
 coverage
 
 yarn-error.log
+
+# used when developing locally
+.vscode/
```
[the remark this answers is not in the document]
Comment (endymion1818) [reply .gitignore]: 👍

File: .gitignore [hunk 5]
```diff
@@ -9,4 +9,4 @@ styleguide/*
 dist
 coverage
 
-yarn-error.log
+yarn-error.log
```
Comment (thegrinder) [inline .gitignore]: missing new line at the end here

Comment (endymion1818) [reply .gitignore]: Fixed, thanks

**Pull request merged**